qmxgraph/html.py

```diff
@@ -1,22 +1,108 @@
-def create_image_tag_str(source, width, height):
+import attr
+
+
+asdict = attr.asdict
+
+
+@attr.s(repr=False, slots=True, hash=True)
+class _HtmlChildTypeValidator(object):
```

Comment:
I will rename it.
I used a class to mimic their validator implementations but since I dropped the `__repr__` (this is why the do it like that) I guess you are right about `functools.partial`.